pkgs/tools/misc/ollama/default.nix

```diff
@@ -191,9 +184,9 @@ goBuild ((lib.optionalAttrs enableRocm {
   };
 
   meta = {
-    changelog = "https://github.com/ollama/ollama/releases/tag/v${version}";
```

Comment:
What's the reason of this change?

Reply:
No good reason, I just moved `changelog` down because I prefer `description` to be at the top of `meta`.